### PR TITLE
Feature/uchv 62 who amends

### PR DESCRIPTION
**Pull request by joefitter**

### CODE REVIEW

**Comment by joefitter** [comment]
@gxxm @JoeChapman @easternbloc @bendiggle 👍 's PLZ

**Comment by joefitter** [comment]
don't forget to squash and merge this horrible mess into a nice neat commit 😻

**Comment by gxxm** [comment]
:thumbsup: :thumbsup:

File: .drone.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ pipeline:
     environment:
       - DOCKER_HOST=tcp://127.0.0.1:2375
     commands:
-      - docker build -t ukvi-complaints .
+      - docker build --no-cache -t ukvi-complaints .
```
Comment (easternbloc) [inline .drone.yml]: we should remove the no-cache here

File: .drone.yml [hunk 2]
```diff
@@ -26,7 +26,7 @@ pipeline:
     environment:
       - DOCKER_HOST=tcp://127.0.0.1:2375
     commands:
-      - docker build -f Dockerfile-acceptance -t acceptance .
+      - docker build --no-cache -f Dockerfile-acceptance -t acceptance .
```
Comment (easternbloc) [inline .drone.yml]: and here

**Comment by joefitter** [comment]
@easternbloc 💟

**Pull request merged**